client/src/core/client/admin/routes/Configure/sections/General/DSAMethodOfRedressOptions.tsx

```diff
@@ -0,0 +1,100 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent } from "react";
+import { Field } from "react-final-form";
+
+import { Validator } from "coral-framework/lib/validation";
+import { RadioButton } from "coral-ui/components/v2";
+
+interface Props {
+  validate?: Validator;
+  name: string;
+  disabled: boolean;
+  format?: (value: any, name: string) => any;
+  testIDs?: {
+    on: string;
+    off: string;
+  };
+  className?: string;
+}
+
+export enum DSAMethodOfRedress {
+  None = "NONE",
+  Email = "EMAIL",
+  URL = "URL",
+}
+
+export const parseVal = (v: any, name: string) => {
```

Comment:
Do we need this `parseVal` and `format`? Seems like it works fine without them?

Reply:
it wasn't working fine for me without them, but I can try again